### PR TITLE
Remove optionalTarget - part 4

### PR DESCRIPTION
**Pull request by WestLangley**
As proposed in #12231.

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks thanks!

File: src/math/Ray.js [hunk 1]
```diff
@@ -67,19 +72,26 @@ Object.assign( Ray.prototype, {
 
 	}(),
 
-	closestPointToPoint: function ( point, optionalTarget ) {
+	closestPointToPoint: function ( point, target ) {
+
+		if ( target === undefined ) {
+
+			console.warn( 'THREE.Ray: .closestPointToPoint() target is now required' );
```
Comment (aardgoose) [inline src/math/Ray.js]: @WestLangley 

This triggers in the Points.raycast() testpoint function.

(I haven't looked at other objects raycast methods)

Comment (Mugen87) [reply src/math/Ray.js]: see #13530

Comment (WestLangley) [reply src/math/Ray.js]: Thanks @aardgoose Trying to track these down...

**Pull request merged**